wolfssl/openssl/evp.h

```diff
@@ -280,6 +297,7 @@ WOLFSSL_API int  wolfSSL_EVP_Cipher(WOLFSSL_EVP_CIPHER_CTX* ctx,
                           unsigned char* dst, unsigned char* src,
                           unsigned int len);
 
+WOLFSSL_API const WOLFSSL_EVP_CIPHER* wolfSSL_EVP_get_cipherbynid(int);
```

Comment:
Just a rough idea of test funciton in api.c. Shall I add a function like this?

`* Test function for wolfSSL_EVP_get_cipherbynid.
 *
 * POST: return 1 on success.
 */

 static void test_wolfSSL_EVP_get_cipherbynid(int nid)
 {

   WOLDSSL_EVP_CIPHER *cipher;
   int nid;

   OpenSSL_add_all_algorithms();

   AssertNotNull(cipher = EVP_get_cipherbynid(nid));
 }
`